calico_node/Makefile

```diff
@@ -588,6 +588,17 @@ node-test-at:
 	chmod +rx /tmp/goss && \
 	/tmp/goss --gossfile /tmp/goss.yaml validate'
 
+.PHONY: ci 
+ci:
```

Comment:
This is the same as most of the semaphore target below.

Why don't we make the semaphore target like this:

```
semaphore:
  <do cleanup>
  $(MAKE) ci
```